src/service/scheduler.js

```diff
@@ -37,10 +39,12 @@ export class Scheduler {
       // Root bounds are not important, so we can use the `root:null` for a
       // top-level window.
       root: isIframed(win) ? win.document : null,
-      rootMargin: '250% 31.25%',
-      threshold: 0.001,
+      rootMargin: ROOT_MARGIN,
```

Comment:
Is the 0.001 no longer necessary?

Reply:
I don't think we need it here. I think I accidentally adopted it from the `DisplayObserver` where it was initially important (and which we later changed to 0.5). But from mount point of view - it's ok to load things which we would otherwise consider undisplayed (or partially obscured).